taskfile.yaml

```diff
@@ -437,25 +347,18 @@ tasks:
           CHECKSUM_FILE: "{{.CHECKSUM_FILE}}"
           INCLUDE_PATTERNS: ["{{.OUTPUT_DIR}}"]
 
-  # NOTE: The log-viewer-webui has four different node_modules directories:
+  # NOTE: The webui has four different node_modules directories:
   # * client
   # * server
   # * log-viewer submodule
   # * the top-level one we call "package"
   # This means we have to create four different checksums. To allow tasks which depend on this task
   # to only have to check one checksum file, we concatenate the four checksum files into one.
```

Comment:
Why'd we abandon this?

Reply:
if you were asking why we abandon the merged checksum approach - 

the `deps` of the `webui-node-modules` task validate the individual components' node_modules directories but never merged the checksums into `webui-node-modules.md5`. as a result, any tasks that depend on `webui-node-modules` (, which should have `webui-node-modules.md5` in their `sources`) would not be notified if there is any change in the node_modules and would not re-run

i might have misunderstood the flow. let me know if so